build.gradle

```diff
@@ -3,7 +3,7 @@ ext {
     VERSION = "1.9.5-SNAPSHOT"
     gdxVersion = VERSION
     robovmVersion = "2.2.1-SNAPSHOT"
-    moeVersion = "1.1.0"
+    moeVersion = "1.1.+"
```

Comment:
Would rather have these as fixed values rather than introduce dynamic versions that may break things down the line.